test/src/Makefile

```diff
@@ -19,7 +19,7 @@ $(V): v
 	cd v && git pull
 	cd v && git reset --hard $(VCOMMIT)
 	$(MAKE) -C v
-	$(V) install radare.r2
+	$(V) install radare.r2 || git clone --depth=1 https://github.com/radare/v-r2 ~/.vmodules/radare/r2
```

Comment:
why v install fails?

Reply:
no idea ... https://vpm.best/jsmod/radare.r2 is giving me:
![404](https://user-images.githubusercontent.com/12002672/74931717-2df36f00-541b-11ea-96fb-b9b9ce3caae0.PNG)
but the server is actually returning 200.